webclient/src/components/views/Login.vue

```diff
@@ -26,8 +26,8 @@
           <router-link to='/createaccount'> Create one now. </router-link>
         </div>
         <div class='menu-bar'>
-          <button class='menu-button' @click="Login" :disabled="!valid"> Login </button>
-          <router-link to='/'> Cancel </router-link>          
+          <button class='menu-button' @click="Login" v-on:click="$root.loading = true" :disabled="!valid"> Login </button>
```

Comment:
let's actually remove these, it feels kind of weird in the end. Just for the login page.